### DISCUSSION TITLE
Running different external systems in different cpp files

### DISCUSSION THREAD

**Discussion by frutas-fruit** [xissburg/edyn#101] [General]

**Comment by frutas-fruit** [comment]


**Reply by xissburg** [reply]
You just have to create one function that calls both of your external system functions and assign that one function as **the** external system.

**Reply by xissburg** [reply]
Yes that's the correct way of doing it. It is good practice to break your systems down into smaller functions. Then you just call them in the right sequence in the actual external system function.
